app/views/partials/sub_partials/_themes.html.erb

```diff
@@ -2,6 +2,17 @@
   title: "View the #{theme[:obj][:label]} page", 
   class: "card--theme", 
   style: "background-image: url('#{cms_fragment_render(:image, theme[:obj])}')" do %>
+  <% unless theme[:pas_no] == -1 %>
+    <div class="card__ribbon">
+      <i class="card__icon <%= theme[:obj][:slug] %>">
+        <%= render '/partials/svgs/pin.svg' %>
+      </i>
+      <p class="no-margin">
+          <span class="card__number"><%= theme[:pas_no] %></span>
```

Comment:
Some cards that would need to have a ribbon have that missing still, like wdpa and oecm.
Worth checking the pas_figure method in the presenter is not skipping some labels

Reply:
Cannot remember if this ever took place, but I seem to recall working on this carousel a few months ago and adding a delimiter to the numbers. Even then there were cards missing that ribbon, not sure if this was intended. I think they may have even been taken out deliberately.